remote-api/src/main/java/org/alfresco/rest/api/impl/GroupsImpl.java

```diff
@@ -852,7 +887,7 @@ public void deleteGroupMembership(String groupId, String groupMemberId)
 
         // Verify if groupMemberId is member of groupId
         AuthorityType authorityType = AuthorityType.getAuthorityType(groupMemberId);
-        Set<String> parents = authorityService.getContainingAuthorities(AuthorityType.GROUP, groupMemberId, true);

```

Comment:
I'm not sure if this is correct.  I'd say that `authorityType` variable is unnecessary here and could be removed,

Reply:
Right, removed